esphome/components/web_server/__init__.py

```diff
@@ -41,6 +42,7 @@
         cv.GenerateID(CONF_WEB_SERVER_BASE_ID): cv.use_id(
             web_server_base.WebServerBase
         ),
+        cv.Optional(CONF_INCLUDE_INTERNAL): cv.boolean,
```

Comment:
We prefer setting defaults in the schema:
```suggestion
        cv.Optional(CONF_INCLUDE_INTERNAL, default=True): cv.boolean,
```